dist/api_data.py

```diff
@@ -1105,6 +1105,9 @@ def __ge__(self, other):
         configure debug specific behavior on a session. Generally only used for internal testing
         purposes.''',
         type='category', subconfig=[
+        Config('checkpoint_fail_before_turtle_update', 'false', r'''
+            Fail before writing a turtle file at the end of a checkpoint.''',
+            type='boolean'),
```

Comment:
Yes, having more control was one of the goals. But actually the main reason why I ended up putting it under session is because session's reconfigure is much lighter-weight. Connection's reconfigure actually does a *lot* of work, so I was looking for a mechanism that would not result in any other state changes.

Reply:
Thanks for the explanation.